README.md

```diff
@@ -13,6 +13,19 @@
 - [GoogleTest](https://github.com/google/googletest) (optional; bundled) - unit testing
 - Doxygen (optional) - documentation
 
+## How to install DLA-Future with spack
+
+We provide a spack package DLA-Future that can be easily add to your own spack as follows:
+
+`spack repo add spack`
+
+This will add a new repository with namespace `dlaf`.
+
+The command to install the library for example on an HPC cluster will be similar to:
+
+`spack install -v --test=root dla-future ^mkl ^hpx max_cpu_count=256`
```

Comment:
```suggestion
`spack install -v --test=root dla-future ^intel-mkl ^hpx max_cpu_count=256`
```

Are you sure this command works? AFAIK `mkl` is a virtual dependency, so you cannot use it as dependency, instead you have to specify a package providing it.

I think it would be nice to give the user the most basic way to start using our library, and I think this looks like the most basic one (I'm not really sure about `-v`). But, since it is not just a simple `spack install dla-future`, I would evaluate specifying that the `HPX` option `max_cpu_count` must be set accordingly to the platform quickly explaining how.